.github/workflows/build_publish.yml

```diff
@@ -5,17 +5,27 @@ on:
     branches: [main]
   pull_request:
     branches: [main]
-
-env:
-  BAHMNI_VERSION: 0.94
   
 jobs:
-  build:
+  build_publish:
     runs-on: ubuntu-latest
```

Comment:
We could also add a `name` field here so that the name appears on the actions UI.